generate-docs/json/excel_1_2/snippets.yaml

```
@@ -3568,6 +3594,23 @@
         sheet.getUsedRange().format.autofitColumns();
         await context.sync();
     });
+'Excel.Range#showOutlineLevels:member(1)':
+  - |-
+    Excel.run(async (context) => {
+        const sheet = context.workbook.worksheets.getActiveWorksheet();
+
+        // This collapses the entire outline.
+        sheet.showOutlineLevels(1, 1);
+        await context.sync();
+    });
+  - |-
+    Excel.run(async (context) => {
+        const sheet = context.workbook.worksheets.getActiveWorksheet();
+
+        // This shows the top 3 outline levels; collapsing any additional sublevels
```

small thing: should end with a period
```suggestion
        // This shows the top 3 outline levels; collapsing any additional sublevels.
```

---

Good catch. I'll update it in the source.